<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matrix Guard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scroll_area">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scroll_area_content">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>904</width>
         <height>717</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="button_switch_row">
        <property name="text">
         <string>Switch rows</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_scalar_row">
        <property name="text">
         <string>Scalar Row</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_add_row">
        <property name="text">
         <string>Add Row</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_scalar_all">
        <property name="text">
         <string>Scalar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
